src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -216,6 +216,86 @@ g.test('mipLevelCount,bound_check,bigger_than_integer_bit_width')
     });
   });
 
+g.test('sampleCount,various_sampleCount_with_all_formats')
+  .desc(`Test texture creation with various (valid or invalid) sample count and all formats`)
+  .subcases(() =>
+    params()
+      .combine(poptions('sampleCount', [1, 2, 4, 8, 16, 32, 256]))
```

Comment:
Should test 0 as well, which is an invalid sampleCount.

Reply:
Done.